ebml_matroska.xml

```diff
@@ -894,7 +896,9 @@ The value of this field should be in the -180 to 180 degree range.</documentatio
 
 Value represents a counter-clockwise rotation, in degrees, around the right vector. This rotation must be applied
 after the `ProjectionPoseYaw` rotation and before the `ProjectionPoseRoll` rotation.
-The value of this field should be in the -90 to 90 degree range.</documentation>
+The value of this field should be in the -90 to 90 degree range.
+
+Setting `ProjectionPosePitch` to 180.0 degrees flips the image vertically.</documentation>
```

Comment:
This is… confusing. We give the expected range and, at the same moment, tell the user to use a value completely outside the range.

Apart from that the same two comments as above (why only positive, not negative, and `180` instead of `180.0`).

Reply:
Yeah, that confuses me too.

Reply:
Indeed, I didn't notice that. I'll check where this -90,90 limit comes from.

Reply:
replaced the flip vertical with a ProjectionPoseRoll+ProjectionPoseYaw operation.